python/neutron.cpp

```diff
@@ -21,13 +21,13 @@ template <class T> void bind_positions(py::module &m) {
     :return: A variable containing the detector pixel positions.
     :rtype: Variable)");
 
-  m.def("source_position", py::overload_cast<T>(source_position), R"(
+  m.def("source-position", py::overload_cast<T>(source_position), R"(
```

Comment:
```suggestion
  m.def("source_position", py::overload_cast<T>(source_position), R"(
```
Can't have hyphen in function name